cmd/up.go

```diff
@@ -96,10 +104,71 @@ func runUp(_ *cobra.Command, _ []string) error {
 	return nil
 }
 
+func makeBasicAuthFiles() error {
+	wd, _ := os.Getwd()
+	pwdFile := wd + "/basic-auth-password"
+	authPassword, err := password.Generate(63, 10, 0, false, true)
+
+	if err != nil {
+		return err
+	}
+
+	err = makeFile(pwdFile, authPassword)
+	if err != nil {
+		return err
+	}
+
+	userFile := wd + "/basic-auth-user"
+	err = makeFile(userFile, "admin")
+	if err != nil {
+		return err
+	}
+
+	return nil
+}
+
+func makeFile(filePath, fileContents string) error {
+	_, err := os.Stat(filePath)
+	if err == nil {
+		log.Printf("file %s exists", filePath)
+		return nil
+	} else if os.IsNotExist(err) {
+		log.Printf("file %s not exists, creating", filePath)
```

Comment:
"file x not exists" -> probably can omit this log, but would be "x doesn't exist" or "file x doesn't exist" instead of "not exists"

Thoughts on removing the log statements?

Reply:
Its useful for users if they are running from a random location and they dont know what file is used/created? Ill change the wording to be more readable